### ISSUE TITLE
we get 84% top-1 in depth18 use simclr2 when Finetuning

### ISSUE THREAD

**Issue by feifaxiaoming** [google-research/simclr#47]
i use cifar10 to train simclrv2 model and depth set 18 , when i trained 1000 epoch ,i finetune my model ,i get 84% top-1  ,its loss in simclrv1 91% top-1  ,it  correct?

**Comment by chentingpc** [comment]
we made some changes to the simclrv1 codebase in 244e7128004c5fd3c7805cf3135c79baa6c3bb96, which has not been tested under CIFAR10 and may lead to this discrepancy. Can you try if the same run (same configuration etc.) with the codebase before the change would produce the same result?

**Comment by feifaxiaoming** [comment]
yes ,i use simclrv1 test is correct with your test result is 91%  ,  i use simclrv2 test is loss than simclrv1  use you inprove code.

**Comment by chentingpc** [comment]
Did you set `--ft_proj_selector=0` when you fine-tuned (this has been set as default in newest code) when you use simclrv2? I don't think there should be a big difference between simclrv1 or simclrv2 if you're using the default architecture etc.

**Comment by feifaxiaoming** [comment]
i use you inprove code  ,i don not change the code.and test on simclrv2

**Comment by chentingpc** [comment]
I think the issue has been fixed now (the `ft_proj_selector` is set to 0 by default now). let me know if you still have the same issue.

**Comment by chentingpc** [comment]
close it for now. please reopen if the issue persists
